src/components/ContentNode/SectionTitle.vue

```diff
@@ -0,0 +1,86 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <component
+   :id="!isTargetIDE ? id : null"
+   :is="tag"
+   class="section-title"
+  >
+    <a
+      v-if="anchor && !isTargetIDE"
+      :href="`#${anchor}`"
+      class="header-anchor"
+      aria-label="hidden"
+      @click="handleFocusAndScroll(anchor)"
+    >#</a>
+    <slot />
+  </component>
+</template>
+
+<script>
+import scrollToElement from 'docc-render/mixins/scrollToElement';
+
+export default {
+  name: 'SectionTitle',
```

Comment:
I think we might want to rename this to better indicate that this component is used for headings. Maybe just `Heading` or `LinkableHeading`? That would also more closely align with the node kind from the Render Node spec (not necessary but maybe nice).

Reply:
I liked it! I renamed it to `LinkableHeading` :)